src/mtinfo.f

```diff
@@ -72,6 +70,4 @@ RECURSIVE SUBROUTINE MTINFO ( CMTDIR, LUNMT1, LUNMT2 )
 C  -----
 
       RETURN
-900   BORT_STR = 'BUFRLIB: MTINFO - BAD INPUT MASTER TABLE DIRECTORY:'
```

Comment:
Is removing these lines related to testing or something else?

Reply:
I was trying to test for this in test_bort.  But I quickly realized that, with the changes to strsuc in #406, it's no longer possible for that routine to return with LMTD<0, which in turn means that lines 75 and 76 can no longer be reached within this routine.

I could have changed the if test to LMTD<=0, but I realized that whatever values get returned for MTDIR and LMTD will always be quickly checked anyway (via an inquire statement inside of subroutine mtfnam), so this check in strsuc is now basically superfluous and can be removed.